### PR TITLE
Fix not logging out when token expires

### PR DESCRIPTION
**Pull request by Georgi2704**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoTable/utils/constants.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const DEFAULT_PAGE_SIZES = [5, 10, 15, 20, 25, 50, 100];
+export const DEFAULT_PAGE_SIZES = [5, 10, 15, 20, 25, 50, 100, 0];
```
Comment (Georgi2704) [inline packages/orchestrator-ui-components/src/components/WfoTable/utils/constants.ts:1]: Also add "Show all rows"

File: packages/orchestrator-ui-components/src/rtk/api.ts [hunk 2]
```diff
@@ -48,6 +53,24 @@ export const handlePromiseErrorWithCallback = <T>(
     });
 };
 
+export const catchErrorResponse: ResponseHandler = async (
+    response: Response,
+) => {
+    if (
+        response.status < HttpStatus.Ok ||
+        response.status >= HttpStatus.MultipleChoices
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/rtk/api.ts:61]: 1st: this predicate is hard to follow ( "smaller than OK" or gte MultipleChoices ) -- consider extracting a predicate that is human readable, like "isNotSuccefsfulResponse".
2nd: in case of an error 401-unauthorized of 403-forbidden, wouldnt that trigger this if-case too? and thus never falling into the else-if and signing out the user?

File: packages/orchestrator-ui-components/src/rtk/api.ts [hunk 3]
```diff
@@ -48,6 +53,24 @@ export const handlePromiseErrorWithCallback = <T>(
...
+    ) {
+        console.error(response.status, response.body);
+    } else if (
+        response.status === HttpStatus.Unauthorized ||
+        response.status === HttpStatus.Forbidden
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/rtk/api.ts:66]: Same as above: consider a human readable predicate.

File: packages/orchestrator-ui-components/src/rtk/api.ts (hunk 2 above)
Comment (Georgi2704) [reply packages/orchestrator-ui-components/src/rtk/api.ts:61]: 2nd: true, nice catch

File: packages/orchestrator-ui-components/src/rtk/api.ts (hunk 3 above)
Comment (Georgi2704) [reply packages/orchestrator-ui-components/src/rtk/api.ts:66]: @ricardovdheijden so a seperate const ? also for the 1st one ?

const isUnauthorized = (status: number) => (status === HttpStatus.Unauthorized || status === HttpStatus.Forbidden)

**Review by DutchBen — APPROVED** [review]

**Review by ricardovdheijden — APPROVED** [review]
